proto/lighthouse-result.proto

```diff
@@ -55,6 +55,8 @@ enum LighthouseError {
   DNS_FAILURE = 19;
   // A timeout in the initial connection to the debugger protocol.
   CRI_TIMEOUT = 20;
+  // The page requested was non-HTML. 
```

Comment:
```suggestion
  // The page requested was not HTML. 
```

Reply:
agh yeah I'll change that, I'll make sure to be consistent next time